db/seeds.d/110-smart_proxy_features.rb

```diff
@@ -1,5 +1,5 @@
 # Proxy features
-proxy_features = ["TFTP", "DNS", "DHCP", "Puppet CA", "BMC", "Realm", "Facts", "Logs", "HTTPBoot", "External IPAM",
+proxy_features = ["Templates", "TFTP", "DNS", "DHCP", "Puppet CA", "BMC", "Realm", "Facts", "Logs", "HTTPBoot", "External IPAM",
```

Comment:
Is this related?

Reply:
yes, It was added in a migration previously instead of the seed.